src/EFCore.PG/Query/ExpressionTranslators/Internal/NpgsqlArrayMethodTranslator.cs

```diff
@@ -17,13 +17,18 @@ namespace Npgsql.EntityFrameworkCore.PostgreSQL.Query.ExpressionTranslators.Inte
     /// </remarks>
     public class NpgsqlArrayMethodTranslator : IMethodCallTranslator
     {
-        static readonly MethodInfo SequenceEqualMethodInfo = typeof(Enumerable).GetTypeInfo().GetDeclaredMethods(nameof(Enumerable.SequenceEqual)).Single(m =>
-                m.IsGenericMethodDefinition &&
-                m.GetParameters().Length == 2);
+        [NotNull] static readonly MethodInfo SequenceEqual =
+            typeof(Enumerable).GetTypeInfo().GetMethods(BindingFlags.Public | BindingFlags.Static | BindingFlags.DeclaredOnly)
+                .Single(m => m.Name == nameof(Enumerable.SequenceEqual) && m.GetParameters().Length == 2);
+
+        [NotNull] static readonly MethodInfo Contains =
+            typeof(Enumerable).GetTypeInfo().GetMethods(BindingFlags.Public | BindingFlags.Static | BindingFlags.DeclaredOnly)
+                .Single(m => m.Name == nameof(Enumerable.Contains) && m.GetParameters().Length == 2);
+
+        [NotNull] static readonly MethodInfo EnumerableAnyWithoutPredicate =
```

Comment:
There is no need to prefix the member with `Enumerable`.

Reply:
This is to distinguish them from Queryable, which also exists. It's a very important distinction in EF Core, you'll see the same prefixing upstream (believe me, I'm the extreme one when it comes to making names terse :))